components/raindrop/actions/create-collection/create-collection.mjs

```diff
@@ -0,0 +1,73 @@
+import raindrop from "../../raindrop.app.mjs";
+
+export default {
+  key: "raindrop-create-collection",
+  name: "Create New Collection",
+  description: "Creates an additional collection",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    raindrop,
+    title: {
+      propDefinition: [
+        raindrop,
+        "title",
+      ],
+    },
+    sort: {
+      propDefinition: [
+        raindrop,
+        "sort",
+      ],
+    },
+    public: {
+      propDefinition: [
+        raindrop,
+        "public",
+      ],
+    },
+    parentID: {
+      propDefinition: [
+        raindrop,
+        "collectionID",
+      ],
+      optional: true,
+      label: "Parent ID",
+      description: "The ID of parent collection. Empty for root collections",
+      withLabel: true,
+    },
+    view: {
+      propDefinition: [
+        raindrop,
+        "view",
+      ],
+    },
+    cover: {
+      propDefinition: [
+        raindrop,
+        "cover",
+      ],
+    },
+  },
+  async run() {
+    const title = this.title?.value ?? this.title;
```

Comment:
You don't need to fallback this way just do
```suggestion
    const title = this.title;
```
or even better you can just deconstruct
```suggestion
    const { title } = this;
```
this comment also applies for the rest of components I saw you are doing the same.